### PR TITLE
feat: add dynamic og images

### PR DESCRIPTION
**Pull request by harish-sethuraman**
add dynamic og image

### CODE REVIEW

File: apps/web/src/lib/api/getMarkup.tsx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  const image = profile
+    ? `https://ik.imagekit.io/lensterimg/tr:n-avatar/${getIPFSLink(
+        profile?.picture?.original?.url ??
+          profile?.picture?.uri ??
+          `https://avatar.tobi.sh/${profile?.ownedBy}_${profile?.handle}.png`
+      )}`
+    : 'https://assets.lenster.xyz/images/og/logo.jpeg';
+
+  const { totalFollowing, totalFollowers } = profile.stats;
+  return (
```
Comment (bigint) [inline apps/web/src/lib/api/getMarkup.tsx:17]: `tw` didn't worked 😕

Comment (harish-sethuraman) [reply apps/web/src/lib/api/getMarkup.tsx:20]: done 🖖🏻

Comment (harish-sethuraman) [reply apps/web/src/lib/api/getMarkup.tsx]: @bigint  move `nFormatter` to common place 😮‍💨 and see preview

File: apps/api/src/lib/api/getProfileMeta.ts [hunk 2]
```diff
@@ -38,24 +43,61 @@ const getProfileMeta = async (req: NextApiRequest, res: NextApiResponse, handle:
...
       return res
-        .setHeader('Content-Type', 'text/html')
+        .setHeader('Content-Type', 'image/png')
         .setHeader('Cache-Control', 's-maxage=86400')
-        .send(generateMeta(title, description, image));
+        .send(pngBuffer);
     }
-  } catch {
+  } catch (error) {
+    console.log('Error: cannot generate og image', error);
```
Comment (shanpriyan) [inline apps/api/src/lib/api/getProfileMeta.ts:100]:
```suggestion
    console.error('Error: cannot generate og image', error);
```

**Pull request closed without merging**